README.md

```diff
@@ -9,40 +9,78 @@ A module to store and interact with blocks.
+  console.log(`BLOCK ${blockNumber}: ${blockHash}`)
+  cb()
+}, (err) => console.log(err || 'Done.'))
+```
+
+# API
 
 - [`Blockchain`](#blockchain)
     - [`new Blockchain(opts)`](#new-blockchainblockdb-detailsdb)
-    - [`BlockChain` Properties](#blockchain-properties)
     - [`BlockChain` methods](#blockchain-methods)
+        - [`blockchain.putGenesis(genesis, [callback])`](#blockchainputgenesisgenesis-callback)
+        - [`blockchain.getHead(name, [callback])`](#blockchaingetheadname-callback)
+        - [`blockchain.putBlocks(blocks, [callback])`](#blockchainputblocksblocks-callback)
         - [`blockchain.putBlock(block, [callback])`](#blockchainputblockblock-callback)
         - [`blockchain.getBlock(hash, [callback])`](#blockchaingetblockhash-callback)
-        - [`blockchain.getBlockInfo(hash, cb)`](#blockchaingetblockinfohash-cb)
-        - [`blockchain.getBlockHashes(parentHash, count, cb)`](#blockchaingetblockhashesparenthash-count-cb)
-        - [`blockchain.getBlockChain(startingHashes, count, cb)`](#blockchaingetblockhashesparenthash-count-cb)
-        - [`blockchain.selectNeededHashes(hashes, cb)`](#blockchainselectneededhasheshashes-cb)
+        - [`blockchain.getBlocks(blockId, maxBlocks, skip, reverse, [cb])`](#blockchaingetblocksblockid-maxblocks-skip-reverse-cb)
+        - [`blockchain.selectNeededHashes(hashes, [callback])`](#blockchainselectneededhasheshashes-callback)
+        - [`blockchain.delBlock(blockHash, [callback])`](#blockchaindelblockblockhash-callback)
+        - [`blockchain.iterator(name, onBlock, [cb])`](#blockchainiteratorname-onblock-cb)        
 
 ## `Blockchain`
 Implements functions for retrieving, manipulating and storing Ethereum's blockchain
 
 ### `new Blockchain(opts)`
 Creates new Blockchain object 
-- `opts.blockDB` - the database where Blocks are stored and retreived by hash. Should be a [levelup](https://github.com/rvagg/node-levelup) instance.
-- `opts.detailsDB` - the database where Block number resolutions and other metadata is stored. Should be a [levelup](https://github.com/rvagg/node-levelup) instance.
+- `opts.db` - Database to store blocks and metadata. Should be a [levelup](https://github.com/rvagg/node-levelup) instance.
 - `opts.validate` - this the flag to validate blocks (e.g. Proof-of-Work).
 
-### `BlockChain` Properties
-- `head` - The Head, the block that has the most weight
-- `parentHead`- The parent of the head block
-- `genesisHash` - The hash of the genesis block
-- `height` - The height of the blockchain
-- `totallDifficulty` - The totall difficulty which is the some of a the difficulty of all the prevous blocks
+[DEPRECATION NOTE]
+The old separated DB constructor parameters `opts.blockDB` and `opts.detailsDB` from before the Geth DB-compatible ``v3.0.0`` release are deprecated and continued usage is discouraged. When provided `opts.blockDB` will be used
+as `opts.db` and `opts.detailsDB` is ignored. On storage level DB formats are not compatible and it is not
```

Comment:
Is this a typo: "On storage level DB formats..."?

Reply:
Made this a bit clearer.